src/script/util/scroll-helpers.ts

```diff
@@ -17,35 +17,36 @@
  *
  */
 
-export function scrollEnd(element) {
+export function scrollEnd(element: HTMLElement) {
   return element.scrollHeight - element.clientHeight;
 }
 
-export function scrollToBottom(element) {
+export function scrollToBottom(element: HTMLElement) {
   element.scrollTop = element.scrollHeight;
   return window.setTimeout(() => {
     if (!isScrolledBottom(element)) {
       return (element.scrollTop = element.scrollHeight);
     }
+    return undefined;
   }, 200);
 }
 
-export function isScrolledBottom(element) {
+export function isScrolledBottom(element: HTMLElement) {
   const scrollTop = Math.ceil(element.scrollTop);
   const scrollHeight = element.scrollHeight;
   const height = element.clientHeight;
   return scrollTop + height >= scrollHeight;
 }
 
-export function isScrolledTop(element) {
+export function isScrolledTop(element: HTMLElement) {
   return element.scrollTop === 0;
 }
 
-export function scrollBy(element, distance) {
+export function scrollBy(element: HTMLElement, distance: number) {
   const scrollTop = element.scrollTop;
   element.scrollTop = scrollTop + distance;
 }
 
-export function isScrollable(element) {
+export function isScrollable(element: HTMLElement) {
   return element.scrollHeight > element.clientHeight;
 }
```

Comment:
Could you please add the return types for each function? Makes it much easier to read :slightly_smiling_face: